libs9s/s9srpcclient.cpp

```diff
@@ -3898,6 +3904,12 @@ S9sRpcClient::registerMySqlReplication(
     jobData["vendor"]           = options->vendor();
     jobData["db_user"]          = options->dbAdminUserName();
     jobData["db_password"]      = options->dbAdminPassword();
+    if(!options->replicationPassword().empty())
```

Comment:
I see duplication of that if statement in few places - can we reduce that using separate method?

Reply:
I am afraid that this param is used on different clusters and its check it is required on their respective registration methods...